examples/simple-lambda-worker/.nvmrc

```diff
@@ -0,0 +1 @@
+14.17.3
```

Comment:
This caught me out because the CDK for this library probably requires node 16, but it's the node version for the example project.